experiements.js

```diff
@@ -0,0 +1,84 @@
+/* boardArr = ['x','','','o','o','x','x','x','o']
+
```

Comment:
I like the experiments file. I always like to see people just trying things out, even if they don't always work.